Bylaws.md

```diff
@@ -157,11 +165,11 @@ The public may follow the Forum’s activities by reading all postings on the Pu
+    For election ballots, each Voting Member is entitled to a single vote regardless of the number of participating Voting Member representatives or whether the Voting Member is categorized as a Certificate Issuer or a Certificate Consumer. Only designated Voting Representatives are allowed to cast votes on behalf of Voting Members. If multiple votes are received from a Voting Member’s representatives, the last vote submitted during the voting period is considered the Voting Member’s vote. Within two (2) weeks after the election ballot closes, the election committee will compile the votes, ensure that only one (1) vote is counted per Voting Member, confirm the results with other members of the election committee, and publish the ballot results by sending an email to the Public Mail List. The election committee will not include any votes submitted before or after the voting period when compiling the votes. The ballot results email will contain only the following information: a short description of the ballot purpose, the total number of votes submitted during the ballot period, and the name of the nominee receiving the most votes. The election committee may include other language as necessary to accurately describe the ballot and any concerns the election committee had with the ballot, provided that such language does not disclose how individual Voting Members voted. The election committee will treat the votes of individual Voting Members as confidential information. The nominee receiving the most votes is appointed to the applicable position, regardless of the number of votes cast and irrespective of whether two thirds (2/3) of the Certificate Issuers or fifty percent (50%) of the Certificate Consumers voted for the nominee. If the election ballot results in a tie among the candidates receiving the most votes, the Chair or Vice Chair will call for another election ballot that includes only the two tying candidates.
```

Comment:
Align with the previous comment, stating `fifty percent (50%) plus one (1) `

```suggestion
    For election ballots, each Voting Member is entitled to a single vote regardless of the number of participating Voting Member representatives or whether the Voting Member is categorized as a Certificate Issuer or a Certificate Consumer. Only designated Voting Representatives are allowed to cast votes on behalf of Voting Members. If multiple votes are received from a Voting Member’s representatives, the last vote submitted during the voting period is considered the Voting Member’s vote. Within two (2) weeks after the election ballot closes, the election committee will compile the votes, ensure that only one (1) vote is counted per Voting Member, confirm the results with other members of the election committee, and publish the ballot results by sending an email to the Public Mail List. The election committee will not include any votes submitted before or after the voting period when compiling the votes. The ballot results email will contain only the following information: a short description of the ballot purpose, the total number of votes submitted during the ballot period, and the name of the nominee receiving the most votes. The election committee may include other language as necessary to accurately describe the ballot and any concerns the election committee had with the ballot, provided that such language does not disclose how individual Voting Members voted. The election committee will treat the votes of individual Voting Members as confidential information. The nominee receiving the most votes is appointed to the applicable position, regardless of the number of votes cast and irrespective of whether two thirds (2/3) of the Certificate Issuers or fifty percent (50%) plus one (1) of the Certificate Consumers voted for the nominee. If the election ballot results in a tie among the candidates receiving the most votes, the Chair or Vice Chair will call for another election ballot that includes only the two tying candidates.
```

Reply:
Similarly, as in https://github.com/cabforum/forum/pull/32#discussion_r1162666897